app/src/main/java/comp3350/mealbuddy/objects/MicroFactory.java

```diff
@@ -0,0 +1,25 @@
+package comp3350.mealbuddy.objects;
+
+import java.io.InterruptedIOException;
+import java.util.EnumMap;
+import java.util.Map;
+
+public class MicroFactory {
+
+    public enum Micros{
+        Iron, Zinc, VitaminA, VitaminB12, VitaminC, Choline
+    }
+
+    private MicroFactory(){}    //dont want to instantiate
```

Comment:
then make it abstract

Reply:
right I forgot about that.  Idk what would be better.  I'm pretty sure the Java Math class just uses a private constructor.  DOesn't really matter here since i got rid of the class